cmd/frontend/internal/frontend/service.go

```diff
@@ -288,95 +293,79 @@ func (fes *FrontEndService) Monitor(ctx context.Context, errCh chan<- error) {
 				delay = 1 * time.Second
 			}
 
-			forced := logForced() // force status logs after config updates
 			protocolOut, err := fes.routingService.ShowProtocolSessions(ctx, lp, `NBR-*`)
 			if err != nil {
-				logger.Info("protocol output", "err", err, "out", strings.Split(protocolOut, "\n"))
-				//Note: if birdc is not yet running, no need to bail out
-				//linkCh <- "Failed to fetch protocol status"
-			} else if strings.Contains(protocolOut, bird.NoProtocolsLog) {
-				if !noConnectivity || init {
-					if refreshCancel != nil {
-						refreshCancel()
-					}
-					_ = denounceFrontend(fes.targetRegistryClient)
-					noConnectivity = true
-					connectivityMap = map[string]bool{}
-					logger.Info("protocol output", "out", protocolOut)
-					//linkCh <- "No protocols match"
+				logger.Error(err, "protocol output", "out", strings.Split(protocolOut, "\n"))
+				denounce = true
```

Comment:
I will add a counter for the "session errors" and denounce after X tries (X configurable)

Reply:
Btw, whenever the code deems denounce is required, the VIPs must be denounced as well. (Unless the FE is terminating/restarting).
Otherwise, for example if the connection is still UP according to bird, then BGP will still attract traffic.

Similarly, health.SetServingStatus should be adjusted.